src/libs/antares/study/include/antares/study/study.h

```diff
@@ -337,17 +341,10 @@ class Study: public Yuni::NonCopyable<Study>, public LayerData
     /*!
     ** \brief Destroy all data of the TS generator '@TS'
     */
-    template<enum TimeSeriesType TS>
-    void destroyTSGeneratorData();
-
-    //! Destroy all data of the load TS generator
-    void destroyAllLoadTSGeneratorData();
-    //! Destroy all data of the solar TS generator
-    void destroyAllSolarTSGeneratorData();
-    //! Destroy all data of the wind TS generator
-    void destroyAllWindTSGeneratorData();
-    //! Destroy all data of the hydro TS generator
-    void destroyAllHydroTSGeneratorData();
```

Comment:
While I agree this code isnt' the greatest, TS generation is another matter. May I suggest you open another PR for that small refactor ?